packages/daemon/__tests__/integration/balances.test.ts

```diff
@@ -34,9 +34,14 @@ jest.mock('../../src/config', () => {
   };
 });
 
-jest.mock('@wallet-service/common/src/utils/index.utils', () => ({
-  assertEnvVariablesExistence: jest.fn(),
-}));
+jest.mock('@wallet-service/common', () => {
+  const originalModule = jest.requireActual('@wallet-service/common');
+
+  return {
+    ...originalModule,
+    assertEnvVariablesExistence: jest.fn(),
+  };
```

Comment:
thought(non-blocking): There's an argument to only building explicitly what is being tested with these mocks, instead of using them whole and just changing some parts. However, this adds complexity to the testing phase when we're testing an entire module.

I don't think we should change this PR on this matter, but it's good to keep in mind that the least we implement on these mocks, the better our tests tend to be. Do you agree?

Reply:
We actually don't need this mock anymore, as we removed it from the common module

e5988056b9ace7762f1728bfd1d8811da71a6f25